src/microbe_stage/Microbe.cs

```diff
@@ -1427,22 +1427,39 @@ private void OnIGotAddedToColony()
             return;
 
         var oldRotation = Rotation;
-        var vectorToParent = GlobalTransform.origin - ColonyParent.GlobalTransform.origin;
-        ChangeNodeParent(ColonyParent);
 
-        var vectorToParentRotated = vectorToParent.Rotated(Vector3.Down, Rotation.y);
-        var vectorToMembrane = Membrane.GetExternalOrganelle(vectorToParentRotated.x, vectorToParentRotated.y);
+        // Gets the global rotation of the parent
+        var globalParentRotation = ColonyParent.GlobalTransform.basis.GetEuler();
+
+        // A vector from the parent to me
+        var vectorFromParent = GlobalTransform.origin - ColonyParent.GlobalTransform.origin;
+
+        // This vector gets as if I had no rotation.
+        // This is important, because GetExternalOrganelle only works with non-rotated microbes
+        var vectorToParentRotated = vectorFromParent.Rotated(Vector3.Up, Rotation.y);
 
-        vectorToParentRotated = (-vectorToParent).Rotated(Vector3.Down, ColonyParent.Rotation.y);
-        var parentVectorToItsMembrane =
-            ColonyParent.Membrane.GetExternalOrganelle(vectorToParentRotated.x, vectorToParentRotated.y);
+        // This vector gets rotated as if the parent had no rotation.
+        var vectorFromParentRotated =
+            vectorFromParent.Rotated(Vector3.Down, globalParentRotation.y);
 
-        var requiredDistance = vectorToMembrane.Length() + parentVectorToItsMembrane.Length();
+        // Calculates the vector from my center to my membrane towards the parent.
+        // This vector gets rotated back, because I've rotated it two calls above.
+        var correctedVectorToParent = Membrane.GetExternalOrganelle(vectorToParentRotated.x, vectorToParentRotated.z)
+            .Rotated(Vector3.Down, Rotation.y);
+
+        // Calculates the vector from the parents' center to his membrane towards me.
+        // This gets added to the vector calculated one call before.
+        correctedVectorToParent += ColonyParent.Membrane
+            .GetExternalOrganelle(vectorFromParentRotated.x, vectorFromParentRotated.z)
+            .Rotated(Vector3.Up, globalParentRotation.y);
+
+        ChangeNodeParent(ColonyParent);
```

Comment:
If this isn't important to the rotation logic, I would move this to the top or the bottom of the method, so that people don't think it's a part of this logic.